ui/packages/catalog-ui-search/src/main/webapp/component/search-form-interactions/search-form-interactions.view.js

```diff
@@ -75,9 +75,11 @@ module.exports =  Marionette.ItemView.extend({
                                         type: 'error'
                                     });
                                     throw new Error('Error Deleting Template: ' + xhr.responseText);                                  
-                                }
-                            }); 
-                        this.removeCachedTemplate(this.model.id);
+                                }.bind(this),
+                                success: function(model, xhr, options) {
+                                    this.options.collectionWrapperModel.deleteCachedTemplateById(this.model.id);
+                                }.bind(this)
```

Comment:
If you need to bind this, you can use a `() => {}` instead.

Reply:
I think I tried lambdas and it didn't work so I immediately fell back to `.bind(this)` and it did work with minimal effort. 

Don't know if I checked if that `options` parameter is equivalent to my `this.options` in the bound context, though, so there's a chance that changing it to `options.collectionWrapperModel` would work just as well

Reply:
Nope turns out they're totally different objects
![image](https://user-images.githubusercontent.com/8922497/40751971-4877f566-6422-11e8-9540-7db87d3e7c11.png)